specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -2629,6 +2629,94 @@
           }
         }
       }
+    },
+	"/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/policies/default": {
```

Comment:
>/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/policies/default [](start = 2, length = 107)

You also need to do GET /providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/policies. Even though there is only one, there are services in Azure that perform automated scans on resources and they have no way of knowing that there's only one and what its name is.

Reply:
This is a singleton resource for a customer and we have been using this pattern as per the conversation with Gaurav. Please let me know If you feel very strongly about adding a GET to list policies for a customer.

Reply:
ok, let's leave as-is